example/example-Tzip16OffChainOne.ts

```diff
@@ -1,7 +1,9 @@
-import { TezosToolkit } from '@taquito/taquito';
```

Comment:
I think the title should be onChain instead of OffChain as the metadata is stored on-chain.

Reply:
i cannot find the file example/contract-origination3.ts it has had it s name changed

Reply:
it must be this one:  example-Tzip16OffChainOne.ts

Reply:
fixed